doc/UsersGuide/source/tables/fv3diagtable.csv

```diff
@@ -0,0 +1,416 @@
+sw_upfx0,Clear sky upward shortwave flux at top of atmosphere - GFS radiation
+lw_upfxc,Total sky upward longwave flux at top of atmosphere - GFS radiation
+lw_upfx0,Clear sky upward longwave flux at top of atmosphere - GFS radiation
+ssrun_acc,Accumulated surface storm water runoff
+evbs_ave,Direct evaporation from bare soil
+evcw_ave,Canopy water evaporation
+snohf,Snow phase change heat flux
+pah_ave,Total precipitation advected heat
+trans_ave,Transpiration
+sbsno_ave,Sublimation (evaporation from snow)
+snowc_ave,Snow cover - GFS lsm
+snowc,Snow cover
+soilm,Total column soil moisture content
+tmpmin2m,Min temperature at 2 meters (height)
+tmpmax2m,Max temperature at 2 meters (height)
+dusfc,Surface zonal momentum flux
+dvsfc,Surface meridional momentum flux
+shtfl_ave,Surface sensible heat flux
+lhtfl_ave,Surface latent heat flux
+totprcp_ave,Surface precipitation rate
+totprcpb_ave,Bucket surface precipitation rate
+gflux_ave,Surface ground heat flux
+dlwsfc,Time accumulated downward longwave flux at surface
+ulwsfc,Time accumulated upward longwave flux at surface
+sunsd_acc,Sunshine duration
+watr_acc,Total water runoff
+ecan_acc,Total evaporation of intercepted water
+etran_acc,Total plant transpiration
+edir_acc,Total soil surface evaporation
+wa_acc,Total water storage in aquifer
+pevpr_ave,Averaged potential evaporation rate
+cwork_ave,Cloud work function (valid only with sas)
+u-gwd_ave,Surface zonal gravity wave stress
+v-gwd_ave,Surface meridional gravity wave stress
+psmean,Surface pressure
+cnvprcp_ave,Averaged surface convective precipitation rate
+cnvprcpb_ave,Averaged bucket surface convective precipitation rate
+cnvprcp,Surface convective precipitation rate
+spfhmin2m,Minimum specific humidity
+spfhmax2m,Maximum specific humidity
+u10mmax,Maximum (magnitude) u-wind
+v10mmax,Maximum (magnitude) v-wind
+wind10mmax,Maximum wind speed
+u10max,Hourly maximum (magnitude) u-wind
+v10max,Hourly maximum (magnitude) v-wind
+spd10max,Hourly maximum wind speed
+t02max,Max hourly temperature at 2 meters
+t02min,Min hourly temperature at 2 meters
+rh02max,Max hourly relative humidity at 2 meters
+rh02min,Min hourly relative humidity at 2 meters
+pratemax,Max hourly precipitation rate
+frzr,Accumulated surface freezing rain
+frzrb,Accumulated surface freezing rain in bucket
+frozr,Accumulated surface graupel
+frozrb,Accumulated surface graupel in bucket
+tsnowp,Accumulated surface snow
+tsnowpb,Accumulated surface snow in bucket
+rhonewsn,Precipitation ice density
+rain,Total rain at this time step
+rainc,Convective rain at this time step
+ice,Ice fall at this time step
+snow,Snow fall at this time step
+graupel,Graupel fall at this time step
+totice_ave,Surface ice precipitation rate
+toticeb_ave,Bucket surface ice precipitation rate
+totsnw_ave,Surface snow precipitation rate
+totsnwb_ave,Bucket surface snow precipitation rate
+totgrp_ave,Surface graupel precipitation rate
+totgrpb_ave,Bucket surface graupel precipitation rate

```

Comment:
There are 3 undocumented variables after `totgrpb_ave` in lines 1877-1914 of GFS_diagnostics.F90: `sfcdlw`, `htrlw`, and `lwhc`. They should probably be added.